src/test/java/seedu/duke/parser/ParserTest.java

```diff
@@ -13,11 +15,30 @@
 class ParserTest {
 
     @Test
-    void parseTemplate_LrTemplate_returnsTemplateCommand() throws DukeException {
+    void parse_noLine_expectException() {
         Parser p = new Parser();
-        String line = "template rl";
+        String line = "";
+        assertThrows(DukeException.class, () -> p.parse(line));
+    }
+
+    @Test
+    void parseHelp_help_returnsHelpCommand() throws DukeException {
+        Parser p = new Parser();
+        String line = "help";
         Command c = p.parse(line);
-        assertTrue(c instanceof TemplateCommand);
+        assertTrue(c instanceof Command); // change to HelpCommand once done
+    }
+
+    @Test
+    void parseTemplate_Template_returnsTemplateCommand() throws DukeException {
+        Parser p = new Parser();
+        String line = "template ";
+        String[] tmp = {"r", "rl", "rc", "lc"};
```

Comment:
Consider changing the array name to a plural like templates? https://se-education.org/guides/conventions/java/basic.html#naming
```suggestion
        String[] templates = {"r", "rl", "rc", "lc"};
```

Reply:
Good idea 👍